sql/core/src/main/scala/org/apache/spark/sql/execution/python/ArrowWindowPythonEvaluatorFactory.scala

```diff
@@ -149,7 +149,7 @@ class ArrowWindowPythonEvaluatorFactory(
 
     private val inMemoryThreshold = conf.windowExecBufferInMemoryThreshold
     private val spillThreshold = conf.windowExecBufferSpillThreshold
-    private val spillSizeThreshold = conf.windowExecBufferSpillSizeThreshold
+    private val sizeInBytesSpillThreshold = conf.windowExecBufferSpillSizeThreshold
```

Comment:
For now I reuse the spill threshold config for the `ExternalAppendOnlyUnsafeRowArray` in-memory buffer threshold as well. We can add new configs if we want (but it will be a lot of configs).